content/en/blog/_posts/2024-04-17-kubernetes-1.30.md

```diff
@@ -0,0 +1,167 @@
+that make up the Kubernetes releases you rely on. This requires the specialized skills of people
+from all corners of our community, from the code itself to its documentation and project management.
+
+We would like to thank the entire [release team](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.30/release_team.md)
+for the hours spent hard at work to deliver the Kubernetes v1.30 release to our community. The 
+Release Team's membership ranges from first-time shadows to returning team leads with experience 
+forged over several release cycles. A very special thanks goes out our release lead, Kat Cosgrove, 
+for supporting us through a successful release cycle, advocating for us, making sure that we could
+all contribute in the best way possible, and challenging us to improve the release process.
+
+### Project Velocity
+
+The CNCF K8s DevStats project aggregates a number of interesting data points related to the velocity of Kubernetes and various sub-projects. This includes everything from individual contributions to the number of companies that are contributing and is an illustration of the depth and breadth of effort that goes into evolving this ecosystem.
+
+In the v1.30 release cycle, which ran for 14 weeks (January 8 to April 17), we saw contributions from [888 companies](https://k8s.devstats.cncf.io/d/9/companies-table?orgId=1&var-period_name=v1.29.0%20-%20now&var-metric=contributions) and [1268 individuals](https://k8s.devstats.cncf.io/d/66/developer-activity-counts-by-companies?orgId=1&var-period_name=v1.29.0%20-%20now&var-metric=contributions&var-repogroup_name=Kubernetes&var-repo_name=kubernetes%2Fkubernetes&var-country_name=All&var-companies=All).
+
+
+### Event Update
+
+<THERE WILL BE ALWAYS A KUBECON/CLOUDNATIVECON AROUND, GIVE THE LATEST INFORMATION>
+
+### Upcoming Release Webinar
+
+<RELEASE WEBINARE WILL TAKE PLACE NORMALLY 30 DAYS AFTER RELEASE, ALIGN WITH CNCF TO HIGHLIGHT THE
+WEBINAR>
+
+Join members of the Kubernetes v1.30 release team on DATE AND TIME TBA to learn about the major
+features of this release, as well as deprecations and removals to help plan for upgrades. For more
+information and registration, visit the event page on the CNCF Online Programs site.
+
+### Get Involved
+<THIS COMMUNITY LIVES BY ITS GREAT COMMUNITY, GET THEM INVOLVED!> The simplest way to get involved
+with Kubernetes is by joining one of the many [Special Interest
+Groups](https://github.com/kubernetes/community/blob/master/sig-list.md) (SIGs) that align with your
+interests. Have something you’d like to broadcast to the Kubernetes community? Share your voice at
+our weekly [community meeting](https://github.com/kubernetes/community/tree/master/communication),
+and through the channels below. Thank you for your continued feedback and support.
+
+- Follow us on Twitter [@Kubernetesio](https://twitter.com/kubernetesio) for latest updates
```

Comment:
💭 Could we consider updating the reference to "𝕏" , the current rebranded name of Twitter?

```suggestion
- Follow us on 𝕏 [@Kubernetesio](https://twitter.com/kubernetesio) for latest updates
```